index.md

```diff
@@ -72,3 +72,83 @@ Once operational, Helen finds a verification agent, Lynn, who was able to inspec
 ## UC15 Verified Impact
 
 Lynn is a verification agent for REC2022. She is called by HydroElec to audit their new production facility. After reviewing the certified proposal, she visits the HydroElec site to inspect and verify both the capacity of the system under operating conditions and the security of the automated smart meters. She uses a standard set of criteria established by the UN to record her observations and make a determination about the project’s compliance with both its certified proposal and UN requirements. After finding everything in order, Lynn issues a verification credential with a six month expiry and a maximum capacity of 20 billion kWh, which she gives to HydroElec. The credential references both the original proposal and its certification using machine-verifiable identifiers to ensure the authenticity of the verification.
+
+
+# Focal Use Cases
+
+## Background
+
+As pressure mounts to balance the yearly global carbon budget due to increasing risk to life, property and markets due to climate related.
```

Comment:
Suggest: Pressure mounts to balance the yearly global carbon budget due to increased risk to life, property and markets because of climate-related disasters/events.